src/samplers/aliastable.jl

```diff
@@ -1,22 +1,8 @@
+using AliasTables
```

Comment:
Can you move this to the joint list of other imports in `src/Distributions.jl`? And maybe just use `import AliasTables` so it's always clear when we refer to something from AliasTables?

Reply:
Yes: 1c3530ce040119d42a0ae057def22f9538f2acd8